public/css/main.css

```diff
@@ -38,3 +39,21 @@ textarea {
 
 .progress {
   display: none; }
+
+.indcbtn {
+  margin: 1px; }
+
+#herobar {
```

Comment:
Duplication here and in `public/css/main.scss` - pick a format (`css`, `scss`, `sass`) and stick to it! 

I am firmly of the opinion that SASS is _the best_.